.github/workflows/test.yml

```diff
@@ -46,7 +46,7 @@ jobs:
               TEST_CATEGORY="not test_stages.py and not test_assemblers.py"
           fi
           OSBUILD_TEST_STORE="${{ env.OSBUILD_TEST_STORE }}" \
-          tox -e "${{ matrix.environment }}" -- $TEST_WORKERS -k "$TEST_CATEGORY"
+          tox -e "${{ matrix.environment }}" -- $TEST_WORKERS -k "$TEST_CATEGORY" --error-on-skip-unless="no SELinux"
```

Comment:
@achilleas-k suggested to use `--ok-to-skip` to avoid the double negative, I like it so unless someone has a better suggestion I would say let's go with that